docs/TESTING.md

```diff
@@ -36,22 +28,61 @@ We're supporting Android and iOS mobile operating systems.
 
 #### Scope
 
-In manual testing, we should focus on all user interface elements, hand washing animation in "Protect Yourself".
+In manual testing, we should focus on all user interface interactions:
+
+* Protect yourself
+* Your questions answered
```

Comment:
Called Q&A now

Reply:
@advayDev1 I've taken those names from Product Design doc for v1.0. Do we have the wrong name there?